### ISSUE TITLE
Mounted Drive's Folders and Files Not Detected

### ISSUE THREAD

**Issue by darrickteng** [jlesage/docker-crashplan-pro#2]
I downloaded v1.0.1 from the synology docker registry. I created a new container and went through the steps.

I mounted the volume (it shows all the sub folders and files when setting up the docker container)
![1](https://user-images.githubusercontent.com/16842318/30759711-0d120b0c-9f8c-11e7-9d5a-c258122fb269.png)
![2](https://user-images.githubusercontent.com/16842318/30759715-0f24925c-9f8c-11e7-8660-e514c6f9abfc.png)

But when I access the GUI and select the folders to back up, the subfolders and files don't exist. As you can see from the screenshot, "Shared Folders" is expanded, but there is no folders or files under it
![3](https://user-images.githubusercontent.com/16842318/30759740-2c3eb778-9f8c-11e7-8d02-17b509edbf88.png)

If there is anything else I can provide to help resolve this issue, please let me know

**Comment by pabloa** [comment]
Try removing the space. And pay attention to the letter case. Try for example using paths on lowercase with no space to mount the volume to backup. Example ```/mnt/folders```

**Comment by darrickteng** [comment]
![4](https://user-images.githubusercontent.com/16842318/30762046-4e695714-9f95-11e7-910d-8f1850d6181d.png)
The problem is still there with all lowercase and no space

**Comment by jlesage** [comment]
Maybe a permission issue?  To avoid problems, make sure the IDs defined by environment variables USER_ID and GROUP_ID match the ones of your shared folder.

**Comment by darrickteng** [comment]
That might have been the issue. I missed the part about running the command to get the output, so I left it undefined.

I made a new container and populated it with the respective uid and gid, but now get a "Server disconnected (Code: 1006)" error when trying to access the web GUI.

The admin account belongs to both the admin and user group, but the gid shown is for the users group. I think it is related to the permission issue. I've tried replacing the gid with the admin group id, but the problem still occurs.

I was able to get rungeict's crash plan pro docker image working. Thanks for the help though! :)
